core/Pos/Core/Txp.hs

```diff
@@ -42,26 +43,27 @@ module Pos.Txp.Core.Types
        -- * Block message
```

Comment:
`-- * Block message`? 🤔 

I bet some changes will be requested eventually, so consider removing this line.